src/main/java/com/code_intelligence/jazzer/mutation/mutator/lang/StringMutatorFactory.java

```diff
@@ -142,14 +144,19 @@ public Optional<SerializingMutator<?>> tryCreate(AnnotatedType type, MutatorFact
         .flatMap(parent -> factory.tryCreate(notNull(asAnnotatedType(byte[].class))))
         .map(byteArrayMutator -> {
           boolean fixUpAscii = type.getDeclaredAnnotation(Ascii.class) != null;
-          return mutateThenMapToImmutable((SerializingMutator<byte[]>) byteArrayMutator, bytes -> {
-            if (fixUpAscii) {
-              fixUpAscii(bytes);
-            } else {
-              fixUpUtf8(bytes);
-            }
-            return new String(bytes, StandardCharsets.UTF_8);
-          }, string -> string.getBytes(StandardCharsets.UTF_8));
+          return mutateThenMapToImmutable((SerializingMutator<byte[]>) byteArrayMutator,
+              bytes
+              -> {
+                if (fixUpAscii) {
+                  fixUpAscii(bytes);
+                } else {
+                  fixUpUtf8(bytes);
+                }
+                return new String(bytes, StandardCharsets.UTF_8);
+              },
+              string
+              -> string.getBytes(StandardCharsets.UTF_8),
+              (Predicate<Debuggable> inCycle) -> "String");
```

Comment:
I guess "better" is subjective here. I kind of liked the more verbose but also more consistent output better. I'm not against this change, but could you explain why you prefer it this way?

Reply:
IMHO the underlying `byte` array is an implementation detail of `String` / the `String` mutator and somehow confused me more than it provided help during debugging. That being said, I'm not really attached to this change. If it may help debugging `String` mutator problems I can gladly revert it.